sql/2018-04-17-subscribers-table.sql

```diff
@@ -0,0 +1,14 @@
+-- This file creates a new table, subscriptions, in the ejabberd DB.
+
+USE ejabberd;
+
+CREATE TABLE subscription (
+    name varchar(50) NOT NULL,
+    subscription varchar(255) NOT NULL,
+    last_updated timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP
+) ENGINE=InnoDB CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+CREATE INDEX i_subscription_name_index USING BTREE on subscription(name(25));
+CREATE INDEX i_subscription_subscription_index USING BTREE on subscription(subscription(25));
+CREATE INDEX i_subscription_last_updated_index on subscription(last_updated);
+CREATE INDEX i_subscription_created_at_index on subscription(created_at);
```

Comment:
@mpope9 indexes can be expensive if they're not used.  You mentioned @zgarbowitz suggested these, I think he should take a quick look so I'll DM him.

Reply:
Oh this must've been a misunderstanding- I think we should have indexes on name and created_at here; I don't think we need them on subscription or last_updated

Reply:
Will change.